CompressImagesFunction/CompressImages.cs

```diff
@@ -92,8 +92,13 @@ public static bool Run(CompressimagesParameters parameters, ICollector<CompressI
 
             try
             {
-                // see if .imgbotconfig exists in repo root
+                // see if .imgbotconfig or .imgbotconfig.json exists in repo root
                 var repoConfigJson = File.ReadAllText(parameters.LocalPath + Path.DirectorySeparatorChar + ".imgbotconfig");
+                if (string.IsNullOrEmpty(repoConfigJson))
```

Comment:
This part will never be reached when `.imgbotconfig` does not exists because `File.ReadAllText` will throw a `FileNotFoundException`. Adding a check if the file exists with `File.Exists` should be added here.

Reply:
made the change by using two variable just to keep myself from repeating things